src/demo/index.ts

```diff
@@ -19,7 +19,7 @@ import { DropFileUserInterface } from './component/DropFileUserInterface';
 
 export * from './helper';
 
-let bpmnVisualization = new BpmnVisualization(window.document.getElementById('graph'));
+let bpmnVisualization: BpmnVisualization;
```

Comment:
if we do it this way, we must assure that the 'bpmnVisualization' is initialized in the exported function 'handleFileSelect' 

another way if someone will try use 'handleFileSelect'  without initializing bpmnVisualization - there will be an error in 
function loadBpmn() {
...
bpmnVisualization.load(bpmn); // bpmnVisualization is undefined
...
}

Reply:
@aibcmars I have proposed something with 088b10d  to avoid such issue. Is that enough?

Reply:
good enough 👍